tests/fixtures/controllers/controllerWithCommonResponses.ts

```diff
@@ -0,0 +1,16 @@
+import { Controller, Get, Response, Post, Route } from '../../../src';
+import { ModelService } from '../services/modelService';
+import { TestModel } from '../testModel';
+
+@Route('Controller')
+@Response('401', 'Unauthorized')
```

Comment:
Let's add a `@Response<Type>('404', 'Not found')` so we can make sure it also works with a type

Reply:
MMmm, please check it, i'm not sure that understand you well.